cli/internal/auth/token.go

```diff
@@ -12,6 +12,15 @@ func tokenNeeded(registry specs.Registry, path string) bool {
 	return registry == specs.RegistryCloudQuery || (registry == specs.RegistryDocker && strings.HasPrefix(path, "docker.cloudquery.io"))
 }
 
+func transformerNeedsToken(sources []*specs.Transformer) bool {
```

Comment:
```suggestion
func transformerNeedsToken(transformers []*specs.Transformer) bool {
```

Reply:
Fixed in [a5c61c1](https://github.com/cloudquery/cloudquery/pull/19166/commits/a5c61c113b44cf4ca767a164e824a9f0e06600a7)